source/plugin/internals/access-transformers.rst

```diff
@@ -0,0 +1,70 @@
+===================
+Access Transformers
+===================
+
+Since some parts of the Minecraft code were not designed to be used from the outside, you may find yourself in a
```

Comment:
It should be noted that access transformers can be used on more than just the Minecraft core codebase.